olm/README.md

```
@@ -19,6 +19,28 @@ This directory packages Agent Sandbox for **OLM** (OperatorHub, OpenShift, and o
+   for crd in \
+       sandboxes.agents.x-k8s.io \
+       sandboxclaims.extensions.agents.x-k8s.io \
+       sandboxtemplates.extensions.agents.x-k8s.io \
+       sandboxwarmpools.extensions.agents.x-k8s.io; do
+     kubectl patch crd "${crd}" --type=json -p='[{"op":"replace","path":"/status/storedVersions","value":["v1beta1"]}]' --subresource=status
+   done
+   ```
+
+> [!WARNING]
+> **InstallPlan Failure Behavior**: If an OLM upgrade is attempted before `storedVersions` is pruned, the Kubernetes apiserver will reject the CRD update and the OLM `InstallPlan` will transition to **Failed**. In OperatorHub UI, this may appear generically as "install failed" (with the root rejection message visible in the `Subscription` status conditions). OLM will **not** automatically retry after rejection—once you patch `storedVersions`, you must re-trigger/approve the `InstallPlan` or reinstall the subscription.
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)